_posts/2021-05-21-js-waku.md

```diff
@@ -0,0 +1,171 @@
+2021-05-21-js-waku.md---
+layout: post
+name:  "Presenting JS-Waku: Waku v2 in the browser"
+title:  "Presenting JS-Waku: Waku v2 in the browser"
+date:   2021-05-21 12:00:00 +0800
+author: franck
+published: true
```

Comment:
Missing image - important for Twitter preview

Screenshot of chat app or similar seems fine